### ISSUE TITLE
Accept Dispute Payout - Data could not be saved. P2SH inner scriptPubKey returned false

### ISSUE THREAD

**Issue by cystic** [OpenBazaar/OpenBazaar-Client#946]
Brief Description: When attempting to Accept Dispute Payout, it give the error: Data could not be saved. P2SH inner scriptPubKey returned false

Operating System (OS and version): Windows 8
OpenBazaar version (shown on `About OpenBazaar` page in menu): 1.0.6
Hardware: 

Reproducible (Always / Almost Always / Sometimes / Rarely / Couldn't Reproduce): 
This is the first time I have gotten the error, but my moderator received the same error while testing an earlier dispute. He was able to resolve his error restarting OpenBazaar, but I have not after several attempts.

Steps to reproduce:
1. Dispute sale
2. Moderator makes ruling
3. Click Accept Dispute Payout.

Observed Behavior:
It won't release funds and there doesn't seem to be a way to back out of the dispute.

Expected Behavior:
Accept Dispute Payout and release funds.

Additional info (links, images, etc go here):

**Comment by jjeffryes** [comment]
@cpacia any thoughts on what that error means?

**Comment by LiteBit** [comment]
I can confirm this with my own case I'm moderating. 

I follow steps to close dispute and I get a confirmation message it has been closed, but the funds never are released. If I go to my page and then back to the case page and click on the dispute I just closed it shows as not closed.

This is an issue I'm facing with a current dispute I'm moderating that should be closed by now. It's not a dispute, the thing was brought to me because the buyer couldn't confirm that he received funds because OB wouldn't allow it. I've talked with both and the dispute is ready to close for 4 days.
